sdcm/nemesis.py

```diff
@@ -1732,10 +1732,13 @@ def repair_nodetool_rebuild(self):
     def disrupt_nodetool_cleanup(self):
         # This fix important when just user profile is run in the test and "keyspace1" doesn't exist.
         test_keyspaces = self.cluster.get_test_keyspaces()
-        for node in self.cluster.nodes:
+
+        def clean_task(node):
             InfoEvent('NodetoolCleanupMonkey %s' % node).publish()
             for keyspace in test_keyspaces:
                 node.run_nodetool(sub_cmd="cleanup", args=keyspace)
+        with ThreadPoolExecutor() as executor:
+            executor.map(clean_task, self.cluster.nodes)
```

Comment:
Where do we wait for those to finish ? where do we get failures from those calls ?

Reply:
Agree, for now it is `fire and run`. We must wait for the result and check it.